requirements.txt

```diff
@@ -7,11 +7,12 @@ azure-core==1.26.3
 azure-identity==1.7.1
 azure-mgmt-core==1.3.2
 azure-mgmt-dns==8.0.0
+azure-mgmt-privatedns==1.0.0
 azure-mgmt-trafficmanager==1.0.0
 certifi==2022.12.7
 cffi==1.15.1
 charset-normalizer==3.0.1
-cryptography==39.0.0
+cryptography==39.0.1
```

Comment:
I assume these updates were pulled in by `./script/update-requirements` and not anything strictly required?

Reply:
exatcly :),  I think I followed all the instructions :)